test/e2e/network/dns.go

```diff
@@ -37,8 +37,9 @@ var _ = SIGDescribe("DNS", func() {
 	f := framework.NewDefaultFramework("dns")
 
 	/*
-	   Testname: dns-for-clusters
-	   Description: Make sure that DNS can resolve the names of clusters.
+		Release : v1.9
+		Testname: DNS, cluster
+		Description: When a Pod is created it MUST be able to resolve cluster dns entries such as kubernetes.default, etc. The Pod MUST validate its hostname entries in the hosts file.
```

Comment:
The first sentence is fine. The second sentence needs to say "MUST be able to" rather than "MUST", but it's also inaccurate beyond that.

"A pod must be able to resolve cluster dns entries such as kubernetes.default both via DNS and via /etc/hosts"

Reply:
Fixed.